Caddyfile-dev

```diff
@@ -0,0 +1,12 @@
+{
+	auto_https off
+}
+
+http://localhost:3000 {
+	reverse_proxy /api* http://strapi:8001
+	reverse_proxy /admin* http://strapi:8001
+	reverse_proxy /i18n* http://strapi:8001
+	reverse_proxy /content-manager* http://strapi:8001
+
+	reverse_proxy http://app:3000
+}
```

Comment:
Should we use the "Subfolder unified" strapi configuration as specified [here in their docs](https://docs.strapi.io/dev-docs/deployment/caddy-proxy) ? This way we don't need to manually enumerate the paths, even if they change in the future. If we don't want the default landing Strapi page, we can use a redirect (https://docs.strapi.io/dev-docs/deployment/caddy-proxy#redirecting-landing-page-to-admin-panel)